src/net/virtual_connection.rs

```diff
@@ -354,68 +358,46 @@ impl VirtualConnection {
                         let stream = self
                             .ordering_system
                             .get_or_create_stream(arranging_header.stream_id());
-
-                        if let Some(packet) =
-                            stream.arrange(arranging_header.arranging_id(), payload)
-                        {
-                            Self::queue_packet(
-                                sender,
-                                packet,
-                                self.remote_address,
-                                header.delivery_guarantee(),
-                                OrderingGuarantee::Ordered(Some(arranging_header.stream_id())),
-                            )?;
-
-                            while let Some(packet) = stream.iter_mut().next() {
-                                Self::queue_packet(
-                                    sender,
-                                    packet,
-                                    self.remote_address,
-                                    header.delivery_guarantee(),
-                                    OrderingGuarantee::Ordered(Some(arranging_header.stream_id())),
-                                )?;
-                            }
-                        }
+                        let address = self.remote_address;
```

Comment:
can you inline this with line 373?

Reply:
nop, `self` is mutable borrowed line 358, I get this error in 373 if I try to inline:
> second borrow occurs due to use of `self` in closure